Jetchat/app/src/main/java/com/example/compose/jetchat/profile/ProfileFragment.kt

```diff
@@ -47,16 +49,15 @@ class ProfileFragment : Fragment() {
 
         return ComposeView(context = requireContext()).apply {
             setContent {
-                viewModel.userData.observeAsState().value.let { userData: ProfileScreenState? ->
+                viewModel.userData.collectAsState().value.let { userData: ProfileScreenState? ->
```

Comment:
Not sure if we should be avoiding using `collectAsState` in this way. This will update (and recompose?) even if the view is in the background.

Reply:
Moved back to LiveData